dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04305UpdateWorkflowActionTable.java

```diff
@@ -188,6 +193,28 @@ private void removeWorkflowActionStepIdWorkflowStepFK() throws DotDataException
         }
     }
 
+    private void executeDropWorkflowActionStepIdIndex(DotConnect dc) throws DotDataException {
+        Logger.info(this, "Removing 'workflow_idx_action_step' index in 'workflow_action' table.");
+        try {
+            dc.executeStatement(dropWorkflowActionStepIdIndexQuery());
+        } catch (SQLException e) {
+            throw new DotRuntimeException(
+                    "An error occurred when removing 'workflow_idx_action_step' index in 'workflow_action' table.",
+                    e);
+        }
+    }
+
+    private void executeDropWorkflowActionStepIdNotNullConstrain(DotConnect dc) throws DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.exception.DotDataException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)